libs/ui-lib/lib/ocm/components/featureSupportLevels/FeatureSupportLevelProvider.tsx

```diff
@@ -26,6 +28,7 @@ export type NewSupportLevelProviderProps = PropsWithChildren<{
   loadingUi: React.ReactNode;
   cluster?: Cluster;
   cpuArchitecture?: string;
+  platformType?: string;
```

Comment:
I'd use better types here as well.

Reply:
@jgyselov we need to close this PR. 
I'll made this type changes in 2.26. Please, open a ticket to do this. Thanks